src/block/carousel/index.php

```diff
@@ -19,3 +19,50 @@ function stackable_load_carousel_frontend_script() {
 	}
 	add_action( 'stackable/carousel/enqueue_scripts', 'stackable_load_carousel_frontend_script' );
 }
+
+if ( ! function_exists( 'stackable_carousel_add_class_images' ) ) {
```

Comment:
```suggestion
// Lazy loaded images inside carousels make the carousel height buggy and show extra
// spaces because the height isn't available in lazy loaded images. Prevent images from
// hidden slides from lazy loading to prevent this.
if ( ! function_exists( 'stackable_carousel_add_class_images' ) ) {
```